fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -20,6 +20,8 @@
       flush_interval 5s
       max_retry_wait 300
       disable_retry_limit true
+      buffer_type "#{ENV['BUFFER_TYPE'] || 'memory'}"
+      buffer_path "#{ENV['FILE_BUFFER_PATH_ES_COPY'] || '/dev/null'}"
```

Comment:
In run.sh, if BUFFER_TYPE is set to file and FILE_BUFFER_PATH is not given, then I was thinking to reset the type to memory.  I noticed that @richm modified the error message as follows.  But I cannot find the "exit" code. 
```
 if [ "$BUFFER_TYPE" = "file" -a -z "$FILE_BUFFER_PATH" ]; then
-    BUFFER_TYPE=memory
-    echo "ERROR: File is set to buffer_type but FILE_BUFFER_PATH is empty.  Using memory buffer"
+    echo "ERROR: BUFFER_TYPE is set to file but FILE_BUFFER_PATH is empty.  Please specify a FILE_BUFFER_PATH.  Exiting."
 fi
```
Shouldn't we call "exit 1" after the "Exiting." error message?